examples/xml-access/xml_access.bal

```diff
@@ -14,25 +14,30 @@ public function main() {
                 <!--Price: $10-->
                 </book>`;
     
-    // You can access child XML items using the field-based or index-based access expressions.
-    io:println(bookXML.author.fname);
-    io:println(bookXML["author"]["fname"]);
+    // You can access child XML items using xml step expressions.
+    io:println(bookXML/<author>/<fname>);
 
-    // Accessing a non-existing child will return `nil`.
-    io:println(bookXML.ISBN.code);
-    io:println(bookXML["ISBN"]["code"]);
-
-    // The result of the above field-based or index-based access is another XML. 
-    // Any XML function can be invoked on top of the resulting XML.
-    io:println(bookXML.author.fname.getTextValue());
-    io:println(bookXML["author"]["fname"].getTextValue());
+    // Accessing a non-existing child will return empty xml sequence.
```

Comment:
```suggestion
    // Accessing a non-existing child will return am empty `xml` sequence.
```
or
```suggestion
    // Accessing a non-existing child will return an empty XML sequence.
```

Reply:
Will apply to https://github.com/ballerina-platform/ballerina-lang/pull/21826